### DISCUSSION TITLE
MPF developer & issues discussions

### DISCUSSION THREAD

**Discussion by toomanybrians** [missionpinball/missionpinball#1] [Announcements]
Hello Fellow Pinball Nerds!

We are planning to move the MPF user support and discussion forums here, to GitHub, which will replace our current Google Groups discussion list.

There are several reasons for this. The biggest is that GitHub's discussion feature (which is what this is that you're reading now) has the modern features we need for a proper support environment. We can classify posts as announcements, questions, bug reports, ideas, polls, feature requests, etc. We can mark answers as "accepted" and build a knowledge base of help. We can have proper code and logfile formatting in posts. We can link and integrate with MPF versions, bug reports, code checkins, etc., (since MPF is hosted on GitHub, there's a ton of cool integrations!)

So, welcome! Let's make some pinball together!
Brian

**Comment by Pucksnok** [comment]
Seems like a wise decision. Just a question, when you shut the Google group down, will it still be available to search or will it disappear completely?
 I learnt plenty from old questions so I hope it still will be available to read, just not post.

**Reply by bzhmaddog** [reply]
I was wondering the same

**Reply by toomanybrians** [reply]
Yeah I’ll just flip the old forum to read-only but keep it there.
